registry.json

```diff
@@ -3898,5 +3898,19 @@
     "telegram": "lukas_psya",
     "operatorName": "",
     "github": "rafek47"
+  },
+  {
+    "ticker": "SHIOK"
+    "poolId": "39d143c3f49b722886983954c83ffe34ec1cb165cf1852d1a88f2f8d",
+    "social": {
+      "website": "https://shiokcardano.github.io",
+      "twitter": "shiokcardano",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```

Comment:
You must add a telegram handle to join the alliance. This must be a TG handle of an individual and not a channel or group